src/shared/scrapers/US/CA/santa-clara-county.js

```diff
@@ -139,45 +143,43 @@ const scraper = {
           method: 'post',
           args: this._urls.postArgs.hospitalized
         });
-        const patients = patientsJSON.results
+        const hospitalized = patientsJSON.results
           .flatMap(r => r.result.data.dsr.DS)
           .flatMap(ds => ds.PH)
           .flatMap(ph => ph.DM0)
           .flatMap(dm0 => dm0.M0)
           .filter(m0 => m0)[0];
 
         currentResult = {
-          date: datetime.getYYYYMMDD(updatedDate),
+          ...currentResult,
           // An accurate time series requires ignoring undated cases.
           cases: totalCases - (process.env.npm_lifecycle_event === 'timeseries' ? undatedCases : 0),
           deaths,
-          hospitalized: patients
+          hospitalized
         };
-        results.push(currentResult);
       }
+
+      // Add tested.
       if (scrapeDateString === datetime.getYYYYMMDD(testUpdatedDate)) {
         // Fetch the current number of tested patients and isolate the number.
         const testsJSON = await fetch.json(this, `${this._urls.query}?tested`, 'tested', false, {
           method: 'post',
           args: this._urls.postArgs.tested
         });
-        const tests = testsJSON.results
+        const tested = testsJSON.results
           .flatMap(r => r.result.data.dsr.DS)
           .flatMap(ds => ds.PH)
           .flatMap(ph => ph.DM0)
           .flatMap(dm0 => dm0.M0)
           .filter(m0 => m0)[0];
-
-        if (currentResult.date === datetime.getYYYYMMDD(testUpdatedDate)) {
-          currentResult.tested = tests;
-        } else {
```

Comment:
If the `scrapeDateString` doesn't match the `updatedDate`, we won't have cases or hospitalized here, so the new code is equivalent to the old.